plugins/search-react/api-report.md

```diff
@@ -199,6 +204,9 @@ export const SearchFilter: {
       SearchFilterComponentProps,
   ): JSX.Element;
   Select(
+    props: Omit<SearchFilterWrapperProps, 'component'> & SelectFilterProps,
```

Comment:
I'm a little confused about having a `Select` and a `MultiSelect`, what would it take to merge the two components and keep only the Select which can be multiple via props?

Reply:
I think if for creating multiselect it would require only to set `multiple` prop then yes, but you will have to override `renderValue` as well, so I don't think it's a good thing to make users of this component to write it every time
```ts
export const MultiSelectFilter = (props: SearchFilterComponentProps) => {
  return (
    <SelectFilter<string[]>
      {...props}
      multiple
      renderValue={selected => selected.join(', ')}
    />
  );
};
```

Reply:
also I'm thinking that if somebody will want to make fully custom multiselect - he has possibility for that

Reply:
Hello @IlyaSavich 

I think what @camilaibs mentions around having one Select component makes sense as that component can be responsible for the multiple select functionality as well, which I think you already have worked on adding. 

To avoid having to add the render value prop every time, maybe we can provide a default render value when the multiple property is passed through? that way the component rendering will look something like

```tsx
 <SearchFilter.Select
        multiple
        label="Search Multi Select Filter"
        name="multi_select_filter"
        values={['value1', 'value2']}
  />
```

and if someone want to override the render values, they can continue to do so by passing both the `multiple` prop as well as `renderValue`

Reply:
makes sense, done